notebooks/Middle_ear_Inter_subject_analysis.ipynb

```diff
@@ -43,45 +43,37 @@
     "import scipy\n",
```

Comment:
<p>This is not the result for number of episodes per hour but rather the power ratio, it should be recomputed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lkorczowski/Tinnitus-n-Sleep/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>